packages/psss/src/index.js

```diff
@@ -4,14 +4,14 @@
  */
 import React from 'react';
 import { render as renderReactApp } from 'react-dom';
-import App from './App';
 import { ReactQueryDevtools } from 'react-query-devtools';
+import App from './App';
 import { AppProviders } from './AppProviders';
-import { initStore } from './store/store';
+import { store } from './store/store';
 
 const render = () => {
   return renderReactApp(
-    <AppProviders store={initStore()}>
```

Comment:
an api class is no longer injected into the store so no longer need to have this be a function